examples/node-app-mssql/connections.sync-db.json.docker

```diff
@@ -0,0 +1,13 @@
+{
+  "connections": [
+    {
+      "id": "db1",
+      "host": "mssql",
+      "port": 1433,
+      "user": "sa",
+      "database": "master",
+      "password": "password",
```

Comment:
Use the same password as you've shown in `README` so that the user can run it w/o making any changes by default. 
```suggestion
      "password": "Password@123",
```

Reply:
Done